.gitignore

```diff
@@ -4,5 +4,7 @@
 !/log/.keep
 /tmp
 /public/uploads
+# Ignore environment variables
+.env
+coverage
 /.env
```

Comment:
You don't need this anymore if you include `.env`. The slash in front means it will only match the `.env` file in the root of the project.